packages/flutter_tools/lib/src/commands/downgrade.dart

```diff
@@ -87,7 +87,7 @@ class DowngradeCommand extends FlutterCommand {
     String workingDirectory = Cache.flutterRoot;
     if (argResults.wasParsed('working-directory')) {
       workingDirectory = stringArg('working-directory');
-      _flutterVersion = FlutterVersion(const SystemClock(), workingDirectory);
+      _flutterVersion = FlutterVersion(clock: const SystemClock(), workingDirectory: workingDirectory);
```

Comment:
you can skip passing the `clock` param, since `SystemClock()` is the default value. This was only explicitly added because it was positional.

Reply:
although, explicitly showing deps is good, too...

Reply:
Ahh, good point